lib/pagerduty/maintenance_client.rb

```diff
@@ -34,7 +36,19 @@ def get_raw(options = {})
         'filter' => 'open',
         'service_ids' => PagerDuty::Configuration.service_ids
       }.merge(options)
-      perform(:get, 'maintenance_windows', query).body
+
+      begin
+        perform(:get, 'maintenance_windows', query).body
+      rescue => e
+        if e.respond_to?(:original_status) && e.original_status == 400
```

Comment:
Probably a nit, but you could condense this conditional to `if e&.original_status == 400`